### ISSUE TITLE
typing manually 0.9999 causes strange behavior

### ISSUE THREAD

**Issue by pablomaurer** [bcherny/format-as-currency#8]
With every 9 i add, it adds a cent.
typing 0.9999 should not be 1.01
typing 0.99999 should not be 1.02

maybe this is interesting for someone http://plnkr.co/edit/8mT3EA?p=preview in this i really like it to see the real value thats behind

**Comment by BuffCoder** [comment]
The reason for this bug due to `.toFixed` rounding up and down.

```
var number = util
          .toFloat(value)
          .toFixed(2)
```

When you type in 0.999 it rounds to 1.00.  Then, typing another 9 is similar to typing 1.009, which rounds to 1.01.  Entering another 9 would be similar to entering 1.019 then, and thus is rounded to 1.02.

I have submitted a [PR](https://github.com/bcherny/format-as-currency/pull/16) to fix this.

**Comment by bcherny** [comment]
fixed in 9b2b850
